src/java/org/apache/cassandra/tools/SSTableMetadataViewer.java

```diff
@@ -321,6 +323,21 @@ private void printSStableMetadata(File file, boolean scan) throws IOException
         CompactionMetadata compaction = statsComponent.compactionMetadata();
         SerializationHeader.Component header = statsComponent.serializationHeader();
         Class<? extends ICompressor> compressorClass = null;
+
+        TableMetadata metadata = Util.metadataFromSSTable(descriptor);
+        SSTableReader sstable = SSTableReader.openNoValidation(null, descriptor, TableMetadataRef.forOfflineTools(metadata));
+        int count = 0;
+        KeyIterator iter = sstable.keyIterator();
+        while (iter.hasNext()) 
```

Comment:
Consider using: 

count = IteratorUtils.size(iterator)

Reply:
it appears apache common collections and guava are not used by the took packages